sympy/core/random.py

```diff
@@ -72,10 +72,13 @@ def verify_numerically(f, g, z=None, tol=1.0e-6, a=2, b=-1, c=3, d=1):
     True
     """
     from sympy.core.symbol import Symbol
+    from sympy.core.sympify import sympify
     from sympy.core.numbers import comp
-    from sympy.core.containers import Tuple
-    f, g, z = Tuple(f, g, z)
-    z = [z] if isinstance(z, Symbol) else (f.free_symbols | g.free_symbols)
+    f, g = (sympify(i) for i in (f, g))
```

Comment:
There is a similar (mis)use of `Tuple` here: https://github.com/sympy/sympy/blob/ed0bfd1e47345c8bd45ef542eef18f6a21c55685/sympy/integrals/integrals.py#L808-L813

Reply:
And here:
https://github.com/sympy/sympy/blob/ed0bfd1e47345c8bd45ef542eef18f6a21c55685/sympy/tensor/indexed.py#L326-L334